report/assets/styles.css

```diff
@@ -695,11 +695,10 @@
   border-top: 1px solid var(--report-border-color-secondary);
 }
 
-.lh-metric:nth-last-child(-n+2) {
+.lh-category:not(.lh--hoisted-meta) .lh-metric:nth-last-child(-n+2) {
```

Comment:
since this is not trivial can you add a comment for why we are doing this? "avoid extraneous border when we move meta block into metrics div" or something